test/test_onnx.py

```diff
@@ -54,6 +55,16 @@ def test_transpose(self):
         torch._C._jit_pass_onnx(trace)
         self.assertONNXExpected(trace.export())
 
+    def test_concat(self):
+        class ConcatNet(nn.Module):
+            def forward(self, inputs):
+                return torch.cat(inputs, 1)
+
+        # volatile is of particular interest; there was a bug involving this
+        x = Variable(torch.randn(2, 3), volatile=True)
+        y = Variable(torch.randn(2, 3), volatile=True)
+        self.assertONNXExpected(export_to_string(ConcatNet(), ((x, y),)))
```

Comment:
no, no, no. Let's figure out a better strategy for testing onnx than dumping 200 expect files (for each op).

Reply:
Can we discuss this in today's standup (i.e., does this block the patch?) I think there are benefits to dumping expect files, at least for some of them.

For this particular patch, I added this test because concat export was segfaulting. So, if it was just the segfault I wanted to test, I could get rid of the assertONNXExpected. But let's talk about it.